### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: MANIFEST [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Changes
+Makefile.PL
+MANIFEST
+README
+t/DBIx-Result-Convert-JSONSchema.t
+lib/DBIx/Result/Convert/JSONSchema.pm
```
Comment (leejo) [inline MANIFEST]: This is incomplete, IIRC you can do make manifest to populate it.

File: Makefile.PL [hunk 2]
```diff
@@ -0,0 +1,11 @@
+use 5.024003;
+use ExtUtils::MakeMaker;
+
+WriteMakefile(
+    NAME          => 'DBIx::Result::Convert::JSONSchema',
+    VERSION_FROM  => 'lib/DBIx/Result/Convert/JSONSchema.pm',
+    PREREQ_PM     => {},
```
Comment (leejo) [inline Makefile.PL]: Need to list non-core pre-reqs and versions here. Probably makes sense to list all modules required, core or non-core, which is what I usually do.

File: README [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Put the correct copyright and licence information here.
+
+Copyright (C) 2018 by Valters Skrupskis
+
+This library is free software; you can redistribute it and/or modify
+it under the same terms as Perl itself, either Perl version 5.24.3 or,
+at your option, any later version of Perl 5 you may have available.
+
+
```
Comment (leejo) [inline README]: You can do `pod2readme path/to/lib/file.pm` and/or `pod2markdown path/to/lib/file.pm` to generate this from the module's perldoc

File: lib/DBIx/Result/Convert/JSONSchema.pm [hunk 4]
```diff
@@ -0,0 +1,458 @@
...
+use warnings;
+
+our $VERSION = '0.01';
+
+use Carp;
+use Module::Load qw/ load /;
+use Readonly;
+
+
+=head1 SYNOPSIS
```
Comment (leejo) [inline lib/DBIx/Result/Convert/JSONSchema.pm]: Probably want NAME, VERSION, and DESCRIPTION pod here too.

**Comment by leejo** [comment]
Add a `.travis.yml` file and we can set up CI. Example: https://github.com/Humanstate/mojolicious-plugin-oauth2-server/blob/master/.travis.yml

**Comment by ghost** [comment]
TODO: Use Moo instead of plain old objects (it's a dependency of DBIx either way...)

File: README.bak [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+Put the correct copyright and licence information here.
+
+Copyright (C) 2018 by Valters Skrupskis
+
+This library is free software; you can redistribute it and/or modify
+it under the same terms as Perl itself, either Perl version 5.24.3 or,
+at your option, any later version of Perl 5 you may have available.
+
+
```
Comment (leejo) [inline README.bak]: Probably want to nuke this and add it to the `.gitignore`

**Pull request merged**